jobs/e2e_node/crio/1.18/crio-install.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+sudo dnf module enable cri-o:1.18 -y
+sudo dnf install cri-o -y
+sudo systemctl start crio
```

Comment:
We could make this platform Independent by sticking to the static binary bundle:
https://github.com/kubernetes/kubernetes/pull/91209/files#diff-6a44976362555568e0243a480da2e86bR518-R520

OTOH, I see the benefit of testing the packages indirectly that way.

Reply:
I was thinking if we test against the released packages that can be installed using `dnf` then not only we test for CRIO but also those packages which get used in the production to install CRIO.

Reply:
SGTM :)